app/controllers/works_controller.rb

```diff
@@ -656,11 +656,7 @@ def show_multiple
     @page_subtitle = ts("Edit Multiple Works")
     @user = current_user
 
-    if params[:pseud_id]
-      @works = Work.joins(:pseuds).where(pseud_id: params[:pseud_id])
-    else
-      @works = Work.joins(pseuds: :user).where('users.id = ?', @user.id)
-    end
+    @works = Work.joins(pseuds: :user).where(users: { id: @user.try(:id) })
```

Comment:
```suggestion
    @works = Work.joins(pseuds: :user).where(users: { id: @user.id })
```
The user can't be nil here because this action is only accessible for logged-in users, so no need for the extra guard with the try.

Reply:
Thanks!

f2551c37b